internal/tiger/mcp/service_tools.go

```diff
@@ -273,11 +334,41 @@ WARNING: Creates billable resources.`,
 		OutputSchema: ServiceCreateOutput{}.Schema(),
 		Annotations: &mcp.ToolAnnotations{
 			DestructiveHint: util.Ptr(false), // Creates resources but doesn't modify existing
-			IdempotentHint:  false,           // Creating with same name would fail
+			IdempotentHint:  false,           // Creating with same name creates multiple services (name is not unique)
 			Title:           "Create Database Service",
 		},
 	}, s.handleServiceCreate)
 
+	// service_fork
+	mcp.AddTool(s.mcpServer, &mcp.Tool{
+		Name:  "service_fork",
+		Title: "Fork Database Service",
+		Description: `Fork an existing database service to create a new independent copy.
+
+You must specify a fork strategy:
+- 'NOW': Fork at the current database state (creates new snapshot or uses WAL replay)
+- 'LAST_SNAPSHOT': Fork at the last existing snapshot (faster fork)
+- 'PITR': Fork at a specific point in time (point-in-time recovery)
+
+By default:
+- Name will be auto-generated as '{source-service-name}-fork'
+- CPU and memory will be inherited from the source service
+- The forked service will be set as the default service
+
+Default behavior: Returns immediately while service provisions in background (recommended).
+Setting wait=true will block for several minutes until ready - only use if user explicitly needs immediate access.
```

Comment:
This wording (after the `-`) is confusing to me. I'd say something more along the lines of "only use if your next steps depends on database readiness to proceed"